### ISSUE TITLE
lists in quotes should not be linted as normal lists?

### ISSUE THREAD

**Issue by mimoo** [DavidAnson/markdownlint#683]
```
src/plonk/lagrange.md:13 MD032/blanks-around-lists Lists should be surrounded by blank lines [Context: "> - $H$: The coset we are in, ..."]
```

the text:

```
> - $H$: The coset we are in, with generator $g$ and offset $h$
> - $m$: The size of the coset $H$
> - $Z_H$: The vanishing polynomial for $H$. $Z_H(x) = \prod_{i \in [m]} (x - h \cdot g^i) = x^m - h^m$
```

a similar rule got triggered on something like this:

```
> 2. thing
> 3. thing
> 4. thing
```

because it didn't start with "1."

**Comment by DavidAnson** [comment]
Lists in blockquotes are valid lists as demonstrated below. Why do you think they should not be linted?

<http://markdown-it.github.io/#md3=%7B%22source%22%3A%22%3E%201.%20Item%5Cn%3E%201.%20Item%5Cn%5CnText%5Cn%5Cn%3E%20-%20Item%5Cn%3E%20-%20Item%5Cn%22%2C%22defaults%22%3A%7B%22html%22%3Afalse%2C%22xhtmlOut%22%3Afalse%2C%22breaks%22%3Afalse%2C%22langPrefix%22%3A%22language-%22%2C%22linkify%22%3Atrue%2C%22typographer%22%3Atrue%2C%22_highlight%22%3Atrue%2C%22_strict%22%3Atrue%2C%22_view%22%3A%22html%22%7D%7D>

**Comment by mimoo** [comment]
because I think it is a valid thing to want to quote parts of a list. For example:

---

Here's my recipe:

1. add eggs
2. add sugar
3. add flour

notice the last two steps:

> 2. add sugar
> 3. add flour

which do not make sense

**Comment by scottamain** [comment]
FYI @mimoo here's how to disable:

```
<!-- markdownlint-disable MD029 -->
> 2. thing
> 3. thing
> 4. thing
<!-- markdownlint-enable MD029 -->
```

**Comment by mimoo** [comment]
mmm, I still disagree because a quote semantically should be able to quote the exact text (and a list is text) so I'm just going to disable that rule globally

**Comment by DavidAnson** [comment]
Great!

**Comment by jeroenheijmans** [comment]
Came here to ask the same thing, the rationale for it being the way it is seems fair enough. Especially with [aforementioned workaround](https://github.com/DavidAnson/markdownlint/issues/683#issuecomment-1361876737). I wanted to add to the conversation with a somewhat more extended example that helped me:

```markdown
# Going through another post

Let's talk about some other post.
We will go through the points in the other post step by step.

<!-- markdownlint-disable MD029 -->

The first thing the other post says is:

> 1. Something incredible

And apparently that is incredible.
But the second point they make is:

> 2. Something less credible

Which we can also talk about at length in non-quoted content.
The linting rule remains disabled.
Even for a third point:

> 3. Something final

Which in this example is the last point in the other post.
So now we can safely enable the linting rule again.

<!-- markdownlint-enable MD029 -->

Which means:

1. Quoted lists above don't give warnings.
2. This list is linted properly
3. The full markdown looks very clean

Which is great!
```

Thanks again for a great plugin. It helps me keep my ramblings _somwehat_ readable!
